cocos/scripting/lua-bindings/manual/network/lua_xml_http_request.cpp

```diff
@@ -1039,8 +1039,9 @@ static int lua_cocos2dx_XMLHttpRequest_getResponseHeader(lua_State* L)
         string value = streamData.str();
         
         
-        auto iter = self->getHttpHeader().find(value);
-        if (iter != self->getHttpHeader().end())
+        map<string, string> httpHeader = self->getHttpHeader();
```

Comment:
`_httpHeader ` member variable should be `std::unordered_map<std::string, std::string>`
and `getHttpHeader` should return `const std::unordered_map<std::string, std::string>&`.
and this line should be 
```c++
const auto& httpHeader = self->getHttpHeader();
```